src/js/views/maps/LayerDetailsView.js

```diff
@@ -118,6 +120,13 @@ define([
           showTitle: false,
           hideIfError: true,
         },
+        {
+          label: "Filter by Property",
+          view: FilterByAttributeView,
+          collapsible: false,
+          showTitle: true,
+          hideIfError: true,
```

Comment:
```suggestion
          hideIfError: true,
          renderIf: (model) => model?.get("filters")?.length > 0,
```

We could add a conditional here to decide whether or not a section should get rendered. Then we can reuse this logic for other sections in the futurel.